repose-aggregator/components/services/akka-http-client-service/akka-http-client-service-impl/src/test/scala/org/openrepose/core/services/serviceclient/akka/impl/AkkaServiceClientImplTest.scala

```diff
@@ -108,8 +108,8 @@ class AkkaServiceClientImplTest extends FunSpec with BeforeAndAfterEach with Mat
     methods.foreach { method =>
       describe(s"using the HTTP request method $method") {
         def akkaServiceClientImplDo(akkaServiceClientImpl: AkkaServiceClientImpl, headers: Map[String, String]): ServiceClientResponse = method match {
-          case "GET" => akkaServiceClientImpl.get(hashKey, uri, headers)
-          case "POST" => akkaServiceClientImpl.post(hashKey, uri, headers, BODY_STRING, MediaType.APPLICATION_XML_TYPE)
+          case "GET" => akkaServiceClientImpl.get(hashKey, uri, headers, true)
```

Comment:
Yes, they should. These were in the commit where I changed the service itself. Can't revert that one, so I'll just manually pull these out.